src/omv/framebuffer.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2013/2014 Ibrahim Abdelkader <i.abdalkader@gmail.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * Framebuffer stuff.
+ *
+ */
+#include "imlib.h"
+#include "omv_boardconfig.h"
+#include "framebuffer.h"
+// If buffer size is bigger than this threshold, the quality is reduced.
+// This is only used for JPEG images sent to the IDE not normal compression.
+#define JPEG_QUALITY_THRESH     (160*120*2)
+
+extern char _fb_base;
+framebuffer_t *fb = (framebuffer_t *) &_fb_base;
```

Comment:
Okay, will go through all code where fb is used and replace with MAIN_FB() which uses a getter.